src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -21,7 +21,9 @@ case class SparkDetail()
 
 case class GangliaDetail()
 
-case class TokenSecret(scope: String, key: String)
+abstract class TokenSecretContainer extends Product with Serializable
```

Comment:
We cannot. It gives the `Unable to find constructor for TokenSecretContainer. This could happen if TokenSecretContainer2 is an interface, or a trait without companion object constructor` error at runtime

Reply:
It's leftovers from previous attempts....They not needed at all now, removed.